### DISCUSSION TITLE
Welcome to CircleType Discussions!

### DISCUSSION THREAD

**Discussion by peterhry** [peterhry/CircleType#137] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MikeM00n** [comment]
I have a context menu that when clicked brings up an option to curve text either up or down. My question is this: is there a way to use circletype.js like this  --> "new CircleType($(this)).radius(curve).refresh();" Where this is the jQuery object being referenced? I seem to be getting back an "undefined" object. Image for reference --> https://imgur.com/vgFeS9z
